lib/Listener/BeforeTemplateRenderedListener.php

```diff
@@ -72,14 +73,18 @@ public function handle(Event $event): void {
 		// FIXME as long as photos app has photos as internal label and appname label
 		if ($response->getRenderAs() === TemplateResponse::RENDER_AS_USER ||
 			 $response->getRenderAs() === TemplateResponse::RENDER_AS_PUBLIC) {
+			$csp = new ContentSecurityPolicy();
 			$this->fixPhotosAppName();
+			$response->setContentSecurityPolicy($csp);
```

Comment:
I doubt that an "empty" Content Security Policy has any effect if no CSP rules are added? Can you explain that?